.github/workflows/lint.yml

```diff
@@ -4,24 +4,24 @@ on:
     branches:
       - main
     paths:
-      - 'Sources/**'
-      - 'Tests/**'
-      - 'test-server/**'
-      - 'Samples/**'
-      - '.github/workflows/lint.yml'
-      - 'scripts/ci-select-xcode.sh'
+      - "Sources/**"
+      - "Tests/**"
+      - "test-server/**"
+      - "Samples/**"
+      - ".github/workflows/lint.yml"
+      - "scripts/ci-select-xcode.sh"
```

Comment:
l: Any practical reason for this change?
Just want to understand.

Reply:
Nope, I added fail-fast below and VSCode changed this.